scripts/switch_brain.py

```
@@ -0,0 +1,679 @@
+
+def get_ngrok_tunnel_url() -> str | None:
+    """Get the public URL of the first running ngrok tunnel."""
+    try:
+        resp = httpx.get(NGROK_API, timeout=3)
+        if resp.status_code != 200:
+            return None
+        data = resp.json()
+        tunnels = data.get("tunnels", [])
+        if tunnels:
+            return tunnels[0].get("public_url", "").rstrip("/")
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)